src/v2/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts

```diff
@@ -1,17 +1,32 @@
 import { paramsToCamelCase } from "v2/Components/ArtworkFilter/Utils/urlBuilder"
 import { initialArtworkFilterState } from "v2/Components/ArtworkFilter/ArtworkFilterContext"
 import { allowedFilters } from "v2/Components/ArtworkFilter/Utils/allowedFilters"
+import {
+  MANUAL_GRID_CURATION_FEATURE_FLAG_KEY,
+  getDefaultSortByFeatureFlag,
+} from "./merchandisingTrial"
 
-export function getWorksForSaleRouteVariables({ artistID }, props) {
+export function getWorksForSaleRouteVariables(
+  { artistID },
+  { location, context }
+) {
+  const featureFlags = context.featureFlags ?? {}
+  const isManualGridCurationEnabled = !!featureFlags[
+    MANUAL_GRID_CURATION_FEATURE_FLAG_KEY
+  ]
```

Comment:
The new shape of `featureFlags` mentioned above will break this. But would also be helpful to know if there is a reason we aren't using the `useFeatureFlags` hook here, rather than checking it from the `context` which I am assuming is the `systemContext` getting passed down? Understanding where the `useFeatureFlags` hook isn't enough for developers use cases will help with its development.

Reply:
@mc-jones - I can quickly answer this! So react hooks can only be used in react components, so they wouldn't work here. Part of the work in [this PR](https://github.com/artsy/force/pull/9545) was ensuring that the flags get passed into our context setup further down, so that when we instantiate the framework router we can pass said context in [here](https://github.com/artsy/force/blob/main/src/v2/System/Router/buildClientApp.tsx#L93) both in `buildClientApp` and `buildServerApp`, so that this data is accessible in all parts of the app, and most notably at the router level, which this file directly touches. 

So the flow is:
- grab feature flags from express
- create context
- inject context into router
- inject context into boot

And then at the router level, we can pluck context values out both in `prepareVariables` [as seen here](https://github.com/artsy/force/blob/main/src/v2/Apps/Auction2/auction2Routes.tsx#L74-L87). Or alternatively in react class components (which can't use hooks) we can access via `this.props.match.context.featureFlags` (which is fortunately rare since most of our app has been migrated to modern react at this point). 

Hope this makes sense!